raven-logback/src/main/java/com/getsentry/raven/logback/SentryAppender.java

```diff
@@ -325,7 +327,7 @@ public void setServerName(String serverName) {
     }
 
     public void setMinLevel(String minLevel) {
-        this.minLevel = Level.toLevel(minLevel, Level.WARN);
```

Comment:
I dropped this default `Level.WARN` because it doesn't make a lot of sense in this usage any more. I wish it would throw an error instead of coercing bad keys to `DEBUG`, but that's how it goes.